buildSrc/src/main/kotlin/org/cqfn/save/buildutils/DockerStackConfiguration.kt

```diff
@@ -109,7 +109,7 @@ fun Project.createStackDeployTask(profile: String) {
                 ""
             } +
                 " save"
-        commandLine("bash", "-c", "'$composeCmd | $stackCmd'")
+        commandLine("bash", "-c", "$composeCmd | $stackCmd")
```

Comment:
This is a hotfix for #547. JavaExec passes these strings directly into `argv`, so single quotes are not executed by bash and cause command to fail.